app.rb

```diff
@@ -0,0 +1,85 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require 'active_record'
+require 'pg'
+
+# Load the file to connect to the DB
+require_relative 'db/connection'
+
+# Load models
+require_relative 'models/pokemon'
+require_relative 'models/trainer'
+
```

Comment:
you could add a get "/" or root route here to redirect to "/pokemons"


 otherwise all else in this looks very 👍